Sources/TuistKit/ProjectEditor/ProjectEditorMapper.swift

```diff
@@ -521,7 +521,8 @@ final class ProjectEditorMapper: ProjectEditorMapping {
             sources: sourcePaths.map { SourceFile(path: $0, compilerFlags: nil) },
             filesGroup: filesGroup,
             dependencies: dependencies,
-            additionalFiles: additionalFilePaths.map { .file(path: $0) }
+            additionalFiles: additionalFilePaths.map { .file(path: $0) },
+            metadata: .init(tags: [])
```

Comment:
Nitpick: I'd replace `.init` with `Type(` to make it easier for the compiler to resolve things.